concepts/pattern-matching-literals/introduction.md

```diff
@@ -51,3 +51,5 @@ closerToZero n
 
 Pattern matching will test a value against each pattern from top to bottom, until it finds a matching pattern and executes the logic associated with that pattern.
 **The order of patterns matters!**
+
+[PM]: http://learnyouahaskell.com/syntax-in-functions#pattern-matching
```

Comment:
In accordance with https://github.com/exercism/haskell/pull/1104, we cannot use http://learnyouahaskell.com anymore. Please update this link to use https://learnyouahaskell.github.io instead.